### DISCUSSION TITLE
Deploy tool

### DISCUSSION THREAD

**Discussion by yeti92force** [YetiForceCompany/YetiForceCRM#16158] [Feature improvement]
Hi dear support,

As customers of Yetiforce CRM, we are looking for a deploy tool to pull standard customisation like fields, profil righ'ts, page layout from our dev organisation to UAT, INT and PROD.

Today, we are doing all configuration manually on each environnement and we would like automate this steps.

Could you provide a standard deployment tool or advise us on a third-party solution to connect our different instances, which would facilitate our deployments?

Thanks.
Regards

**Comment by mariuszkrzaczkowski** [comment]
We do not have a deployment tool,
depending on the changes, these are transferable changes:
- package with module (module creation / update) 
- bundle of files only
- PHP script to upload custom changes to the database
- we always keep all changes in the internal gitlab to version the introduced changes and accept between environments

The model of introducing changes may be different, depending on the needs and implementation stage

**Reply by yeti92force** [reply]
Thanks for your feedback.
Is it possible de have a set of good practices about "bundle of files". 
example : moove specific profiles from one org to a specific org

regards
